pom.xml

```diff
@@ -48,7 +48,7 @@
             (these are downloaded and installed in these versions by Maven for the CI profiles)
          -->
         <payara.version>4.1.2.181</payara.version>
-        <payara.micro.version>5.182</payara.micro.version>
```

Comment:
Note that we plan to replace this property with just payara.version as the two are never needed at the same time

Reply:
Adjusted to `payara.version`